pkg/controller/observe.go

```diff
@@ -40,12 +40,20 @@ func generateObservation(in *release.Release) v1alpha1.ReleaseObservation {
 	if relInfo != nil {
 		o.State = relInfo.Status
 		o.ReleaseDescription = relInfo.Description
+		o.Revision = in.Version
 	}
 	return o
 }
 
 // isUpToDate checks whether desired spec up to date with the observed state for a given release
-func isUpToDate(ctx context.Context, kube client.Client, in *v1alpha1.ReleaseParameters, observed *release.Release, s v1alpha1.ReleaseStatus) (bool, error) {
+func isUpToDate(ctx context.Context, kube client.Client, in *v1alpha1.ReleaseParameters, observed *release.Release, s v1alpha1.ReleaseStatus) (bool, error) { // nolint:gocyclo
```

Comment:
maybe it's time to increase a limit in linter config since we're adding this after every meaningful function here.

Reply:
Good point, could remove 3 of 4 by increasing 10 --> 15